src/features/areas/types.ts

```diff
@@ -10,6 +16,7 @@ export type Visit = {
 
 export type Household = {
   id: string;
+  ratings: Rating[];
   title: string;
   visits: Visit[];
 };
```

Comment:
I think we have room to think about what a "rating" is and what a "visit" is, and how they relate to each other in both the data structure and what they mean in the UI. Not for this PR to solve, just thought about it, since in the conversation we had a bit ago it seemed that you and @Bagera had different thoughts on what a rating was.